x-pack/plugins/apm/public/components/app/service_overview/service_overview_errors_table/index.tsx

```diff
@@ -134,8 +120,8 @@ export function ServiceOverviewErrorsTable({ serviceName }: Props) {
       pageIndex,
       direction,
       field,
-      // not used, but needed to trigger an update when comparisonType is changed either manually by user or when time range is changed
-      comparisonType,
+      // not used, but needed to trigger an update when offset is changed either manually by user or when time range is changed
+      offset,
```

Comment:
now that we are using it in the function, does it mean we no longer have to disable the eslint rule?

Reply:
It doesn't seem to be used above

Reply:
Ah, I meant to comment on `comparisonEnabled` but I see what you mean now.